composer.json

```diff
@@ -83,7 +83,8 @@
         "netgen/layouts-ezplatform": "~1.2.0",
         "netgen/layouts-ezplatform-site-api": "~1.2.0",
         "netgen/layouts-ezplatform-relation-list-query": "^1.2",
-        "netgen/layouts-ezplatform-tags-query": "^1.2"
+        "netgen/layouts-ezplatform-tags-query": "^1.2",
+        "primedigital/translations-bundle": "^1.0"
```

Comment:
Can you move this directly below `netgen/admin-ui-bundle` as I'd like to keep this section separate for Netgen Layouts?

Reply:
Fixed in https://github.com/netgen/media-site/pull/32/commits/30eaaea629b84d9c57c4b41c3f2831f7207b244d